osv/testdata/packagist_test_cases.txt

```diff
@@ -0,0 +1,53 @@
+1 < 2
```

Comment:
Can we add the tests from https://github.com/composer/semver/blob/main/tests/ComparatorTest.php as well?

Reply:
Added now